server/gateway/config.json

```diff
@@ -82,10 +82,8 @@
         "blobStorageUrl": "http://historian:3000",
         "internalBlobStorageUrl": "http://historian:3000",
         "repository": "prague",
-        "npm": "https://pragueauspkn.azureedge.net",
-        "clusterNpm": "https://pragueauspkn.azureedge.net",
-        "localNpm": "http://localhost:3002",
-        "localClusterNpm": "http://auspkn:3000",
+        "npm": "https://unpkg.com",
```

Comment:
We need to restrict in our code loader  the namespace we would load that is controlled to avoid running random code..

Reply:
We could remove the non-iframed routes instead? Or, possibly, in addition?